extensions/quality/json-schema/schema.json

```diff
@@ -0,0 +1,125 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://linz.github.io/STAC/extensions/quality/v1.0.0/schema.json#",
+  "title": "Template Extension",
+  "description": "STAC Template Extension for STAC Items and STAC Collections.",
+  "oneOf": [
+    {
+      "$comment": "This is the schema for STAC Items. Remove this object if this extension only applies to Collections.",
```

Comment:
@l0b0 , maybe all such default comments coming from template in this file should be updated

Reply:
Yes, they will be. I'm just trying to get started with something which is as close as possible to the original template content.